gymnasium/core.py

```diff
@@ -66,6 +66,8 @@ class Env(Generic[ObsType, ActType]):
 
     # Created
     _np_random: np.random.Generator | None = None
+    # will be set to the "invalid" value -1 if the seed of the currently set rng is unknown
+    _np_random_seed: int | None = None
```

Comment:
This would probably be better with one consistent way of specifying an invalid state - either None or -1, not both. Unless there's a clear distinction between what None and -1 mean?

Reply:
Yes, `-1` means unknown seed, until `None` means that the seed hasn't been set

Reply:
The return type is int, like np_random itself, the seed is set if it was None before.

The user will never get None, but we need to instantiate to None in the constructor. So the semantics of None is "not yet set", just like for np_random itself